target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8174-mx4x00.dtsi

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
I think this change is not too complicated and it prevents us from duplicating code.

Reply:
I personally don't think this is a good idea even though I get the point of the duplicated code. It's a departure from the paradigm, from what I can see on any other target, and it makes a lot harder to see what the device tree actually is on any of the devices involved. It's not generic configuration for an include IMHO, it's just 'because it coincides I put it on an include file' but that could be done with many devices.